src/card.component.mjs

```diff
@@ -25,10 +13,17 @@ class DevCard extends HTMLElement{
             ${css}
         </style>
         <span class="devcard-htmlContent">
-            ${htmlContent}
+            <div class="card" style="width: ${this.dataset.width}">
+                <div class="header">
```

Comment:
Hi Looking great. But the thing is doing `style="width:"` will change the width of a `<div class="card">` 
The structure is something like 
```html
<dev-card>
    #shadow-root
    <span class="devcard-htmlContent">
        <div class="card">
```

So it would be ideal to change the width of the `<dev-card>` instead. We can do this with 
```js
this.style.width = this.dataset.width
```
```

Also, currently the css has .card{width:300px} I guess we will have to remove it.

Reply:
I've committed the changes. Check now!